daisy/compute/compute.go

```diff
@@ -273,7 +273,7 @@ type operationGetterFunc func() (*compute.Operation, error)
 
 func (c *client) zoneOperationsWait(project, zone, name string) error {
 	return c.operationsWaitHelper(project, name, func() (op *compute.Operation, err error) {
```

Comment:
No need to wait for 1s in operationsWaitHelper when we use this wait operation.

Reply:
That is still important as Wait can still return early for a number of reasons (see link in description). So we should treat it the same as Get.